packages/auto-approve/src/utils-for-pr-checking.ts

```diff
@@ -244,8 +244,30 @@ export function doesDependencyChangeMatchPRTitle(
   return false;
 }
 
+export function doesDependencyAgainstRegexes(
+  versions: Versions,
+  regexToInclude: RegExp[] | undefined,
+  regexToExclude: RegExp[] | undefined
```

Comment:
These could be optional
```suggestion
  regexToInclude?: RegExp[],
  regexToExclude?: RegExp[]
```